Sources/GRPCHTTP2Core/Client/Connection/ConnectionFactory.swift

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-import NIOCore
-import NIOHTTP2
+package import NIOCore
+package import NIOHTTP2
 import NIOPosix
```

Comment:
Sorry to be a pain, but can we have explicit imports for everything? (Hopefully `sed` can make this an easier fix...!)